router/create.js

```diff
@@ -0,0 +1,41 @@
+import microCors from 'micro-cors';
+
+function createRouter(middlewareHandlers = [], errorHandler) {
+  return function useRouter(routeHandlers = [], options) {
+    const handlers = [...middlewareHandlers, ...routeHandlers];
+    const cors = microCors(options);
+
+    return cors(
+      (request, response) =>
+        new Promise((resolve) => {
+          function execute(currentHandlers) {
+            const handler = currentHandlers[0];
+            const nextHandlers = currentHandlers.slice(1);
+            const isLastHandler = !nextHandlers.length;
+
+            // eslint-disable-next-line consistent-return
+            return async function executeNext(error) {
+              if (response.headersSent) return;
```

Comment:
Não faz sentido a gente usar o return nos response pra evitar esse tipo de erro?
Ao invés de `response.json(...)` a gente usar `return response.json(...)`.

O `return` não é mandatório, mas ajuda a mitigar problemas como esse. 

Threads que podem ajudar: 
- https://stackoverflow.com/questions/16810449/when-to-use-next-and-return-next-in-node-js
  - https://stackoverflow.com/a/43404213
- https://stackoverflow.com/questions/34256978/in-express-js-should-i-return-response-or-not
    - https://stackoverflow.com/a/60098044